CP-329 track_visibility for reference on payment option

simgonzalez opened the pull request.

Commit:
CP-329 track_visibility for reference on payment option

ecino commented on recurring_contract/models/contract_group.py:
```diff
@@ -49,7 +49,7 @@ class ContractGroup(models.Model):
         'res.partner', 'Partner', required=True,
         ondelete='cascade', tracking=True, readonly=False)
-    ref = fields.Char('Reference')
+    ref = fields.Char('Reference', track_visibility='onchange')
```
This is the v12 way.
```suggestion
    ref = fields.Char('Reference', tracking=True)
```

simgonzalez committed:
Update recurring_contract/models/contract_group.py

Le maitre d'Odoo a encore frappé. V.14 tracking=True not track_visibility='onchange'. New version to track change on a model

Co-authored-by: ecino <ecino@users.noreply.github.com>
